libs/ledgerjs/packages/hw-app-aptos/package.json

```
@@ -0,0 +1,60 @@
+{
+  "name": "@ledgerhq/hw-app-aptos",
+  "version": "6.29.4",
+  "description": "Ledger Hardware Wallet Aptos Application API",
+  "keywords": [
+    "Ledger",
+    "LedgerWallet",
+    "algo",
+    "Aptos",
+    "NanoS",
+    "Blue",
+    "Hardware Wallet"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/LedgerHQ/ledger-live.git"
+  },
+  "bugs": {
+    "url": "https://github.com/LedgerHQ/ledger-live/issues"
+  },
+  "homepage": "https://github.com/LedgerHQ/ledger-live/tree/develop/libs/ledgerjs/packages/hw-app-aptosd",
```

there is a typo at the end

```suggestion
  "homepage": "https://github.com/LedgerHQ/ledger-live/tree/develop/libs/ledgerjs/packages/hw-app-aptos",
```